tests/Integration/Behaviour/Features/Context/Domain/Product/Combination/UpdateCombinationStockFeatureContext.php

```diff
@@ -0,0 +1,153 @@
+    /**
+     * @Then combination :combinationReference should have following stock details:
+     *
+     * @param string $combinationReference
+     * @param CombinationStock $expectedStock
+     *
+     * @throws CombinationConstraintException
+     */
+    public function assertStockDetails(string $combinationReference, CombinationStock $expectedStock): void
+    {
+        $actualStock = $this->getCombinationForEditing($combinationReference)->getStock();
+
+        Assert::assertSame(
+            $expectedStock->getQuantity(),
+            $actualStock->getQuantity(),
+            sprintf('Unexpected combination "%s" quantity', $combinationReference)
+        );
+        Assert::assertSame(
+            $expectedStock->getMinimalQuantity(),
+            $actualStock->getMinimalQuantity(),
+            sprintf('Unexpected combination "%s" minimal quantity', $combinationReference)
+        );
+        Assert::assertSame(
+            $expectedStock->getLowStockThreshold(),
+            $actualStock->getLowStockThreshold(),
+            sprintf('Unexpected combination "%s" low stock threshold', $combinationReference)
+        );
+        Assert::assertSame(
+            $expectedStock->getLocation(),
+            $actualStock->getLocation(),
+            sprintf('Unexpected combination "%s" location', $combinationReference)
+        );
+
+        if (null === $expectedStock->getAvailableDate()) {
+            Assert::assertSame(
+                $expectedStock->getAvailableDate(),
+                $actualStock->getAvailableDate(),
+                sprintf('Unexpected combination "%s" availability date. Expected NULL, got "%s"',
+                    $combinationReference,
+                    var_export($actualStock->getAvailableDate())
+                )
+            );
+        } else {
+            Assert::assertEquals(
+                $expectedStock->getAvailableDate()->format(DateTimeUtil::DEFAULT_DATETIME_FORMAT),
+                $actualStock->getAvailableDate()->format(DateTimeUtil::DEFAULT_DATETIME_FORMAT),
+                sprintf('Unexpected combination "%s" availability date', $combinationReference)
+            );
+        }
+    }
+
+    /**
+     * @param UpdateCombinationStockCommand $command
+     * @param array<string, mixed> $dataRows
+     */
+    private function fillCommand(UpdateCombinationStockCommand $command, array $dataRows): void
+    {
+        if (isset($dataRows['quantity'])) {
+            $command->setQuantity((int) $dataRows['quantity']);
+        }
+        if (isset($dataRows['minimal quantity'])) {
+            $command->setMinimalQuantity((int) $dataRows['minimal quantity']);
+        }
+        if (isset($dataRows['location'])) {
+            $command->setLocation($dataRows['location']);
+        }
+        if (isset($dataRows['low stock threshold'])) {
+            $command->setLowStockThreshold((int) $dataRows['low stock threshold']);
+        }
+        if (isset($dataRows['low stock alert is on'])) {
```

Comment:
```suggestion
        if (isset($dataRows['low stock alert is enabled'])) {
```

Should we name this the same way we renamed the getters/setters?